packages/djangoflow_app/lib/src/bloc/app_cubit.dart

```diff
@@ -47,6 +45,12 @@ class AppCubit extends HydratedCubit<AppState> {
         ),
       );
 
+  void toggleLocale(String locale) => emit(
```

Comment:
Hi, thanks again for updating the code. From my understanding, toggle means switching between some specific state properties. Here we are actually not switching between a list of particular states, but instead updating the locale. My suggestion will be to rename it to `updateLocale`/`changeLocale` instead.